### DISCUSSION TITLE
More consistent document path in search results

### DISCUSSION THREAD

**Discussion by iexos** [outline/outline#6220] [Ideas]
In search result, sometimes only the collection name is displayed below the document title/content, sometimes the full path. It seems to depending on if the collection is opened in the sidebar, but sometimes it just displays only the collection name either way. This is confusing.

For me it would be helpful to always see the full path, as the document title and collection name sometimes don't give enough context.

**Comment by tommoor** [comment]
Do you have an example? The breadcrumbs on search results should always have the same format

**Comment by iexos** [comment]
![Screenshot from 2023-11-29 11-25-26](https://github.com/outline/outline/assets/15062278/1f181854-2911-412e-b27c-fe6b5d122f28)
![Screenshot from 2023-11-29 11-25-59](https://github.com/outline/outline/assets/15062278/16f97d88-df07-4efd-a8a6-46fbdc87bd5f)


Steps from first to second screenshot:
1. collapse collection
2. reload

**Reply by tommoor** [reply]
Perfect, thank you. This is indeed a bug.
